### ISSUE TITLE
Worn items that are destroyed should leave behind a nonfunctional "Destroyed X" item, that disintegrates when removed, as a reminder that something was worn there.

### ISSUE THREAD

**Issue by Jordanb716** [CleverRaven/Cataclysm-DDA#29531]
**Is your feature request related to a problem? Please describe.**  
When going through combat quickly I sometimes don't notice when something I'm wearing has been destroyed in the log. It also seems weird to me that a destroyed worn item would just completely disappear without a trace.

**Describe the solution you'd like**  
If an item of clothing that is being worn is destroyed, replace it with a "Destroyed X" variant (e.g. "Destroyed Backpack") that does nothing. Item should disintegrate or fall apart with an appropriate message in the log when removed.

**Describe alternatives you've considered**  
Could have a different name, such as 'shredded'. Destroyed worn items could still have encumbrance instead of being completely nonfunctional.

Might be possible to do procedurally, maybe with a generic 'Destroyed' item that takes the name of the item, maybe even something like a container? Something like how if you fill a bottle it's name becomes "bottle of water", maybe have a container that spawns, and fills itself with the destroyed item to take it's name, so that way each item doesn't have to be individually edited (assuming that's a problem.)

**Comment by echomirage1** [comment]
This is a very good idea. It is not uncommon to go around for a day or so without realizing that I have no pants on. Also the these items could be used as environmental storytelling items without inflating the amount of loot.

**Comment by SourSlime** [comment]
For clothing, it could be a good idea to have the clothing piece removed from the player and dropped on the ground beneath them, as if the clothing is so torn it can no longer be worn/fell off.

**Comment by kevingranade** [comment]
This is a great idea.  We're trying to obey conservation of mass in general
to prevent surprising and nonsensical situations, and this is a good way to
close one more hole.

Possibly the one thing you can do with it is disassemble into rags or
whatever.

**Comment by Zireael07** [comment]
Just a hint, could you rephrase the issue so that it's clear this is a proposal, and not a bug?
